core/tauri-utils/src/lib.rs

```diff
@@ -211,6 +211,108 @@ impl Default for Env {
   }
 }
 
+/// Size unit type. Effective to size related settings
+#[derive(Debug, Clone, PartialEq, Eq)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+pub enum SizeUnit {
+  /// Use logical pixel size
+  Logical,
+  /// Use physical pixel size
+  Physical
+}
+
+impl Default for SizeUnit {
+    fn default() -> Self {
+        SizeUnit::Logical
+    }
+}
+
+impl Serialize for SizeUnit {
```

Comment:
Just derive `Serialize` and `Deserialize` instead of manual implementation

Reply:
Hi, thanks for the thoughtful review, but I notice other enum uses manual impl since the macro device approch need the config value to be capitalized. Eg: you have to set "Physical" instead of "physical" in config file.

Reply:
yeah, you need to also ads `#[serde(rename_all = "camelCase"]`